src/fetch.js

```diff
@@ -0,0 +1,18 @@
+const { checkStatus } = require('./util')
+const fetch = require('node-fetch')
+
+const baseHeaders = (additionalHeaders = {}) => ({
+  'User-Agent': '@prismarineJS/prismarine-realms',
```

Comment:
Could this cause problems or raise any flags on their server? Any reason to not use something standard? There are reports of people getting accounts flagged, let's avoid that

Reply:
Yes potentially, I normally would ignore this but it was present in pauth so was following suit. Maybe we remove it from here and there.